src/custom/api/gnosisProtocol/api.ts

```diff
@@ -266,7 +271,7 @@ async function _handleQuoteResponse(response: Response, params?: FeeQuoteParams)
       // report to sentry
       Sentry.captureException(sentryError, {
         tags: { errorType: 'getFeeQuote' },
-        contexts: { params },
+        contexts: { params: { ...params } },
```

Comment:
Is this not the same as?:
```suggestion
        contexts: { params },
```

Reply:
TS was complaining